frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx

```diff
@@ -39,6 +39,14 @@ type AddQuestionModalProps = {
   isFormTemplatePage?: boolean;
 };
 
+enum OptionErrorStatus {
+  NO_OPTION = "Options cannot be empty",
+  AT_LEAST_ONE = "You must enter at least 1 option",
+  AT_LEAST_TWO = "You must enter at least 2 options",
+  NO_DUPLICATES = "Duplicate options are not allowed",
```

Comment:
Can you change `NO_OPTION` to `EMPTY_OPTION` and `NO_DUPLICATES` to `HAS_DUPLICATES`? I think those are better descriptors.

Reply:
Thanks William! I agree they are better descriptors. I have changed the names and fixed the lint warnings in my latest commit.